utils/docker/docker_entrypoint.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+set -e
+
+
+# files mounted from host system, so need to source here instead of in Dockerfile
+echo "source $HOME/self_balancing_robot/devel/setup.bash" >> $HOME/.bashrc
+echo "source $HOME/self_balancing_robot/src/utils/scripts/setup.bash" >> $HOME/.bashrc
+source $HOME/.bashrc
+
+cd $HOME/self_balancing_robot/
+catkin build
+
+clear
+
+ssh-add ~/.ssh/id_rsa
```

Comment:
This fails for me: I get the error `Could not open a connection to your authentication agent.`
This is because the ssh-agent is not running yet, so the command to add the key fails.
This causes my Docker entrypoint script to fail, and so I can't launch the container.

I looked it up, the solution is to add a line in front starting the ssh agent in the background:
```
eval "$(ssh-agent -s)"   # will start ssh agent in the background
ssh-add ~/.ssh/id_rsa    # now this command will work
```
If you want to suppress output from these commands, you can also add `&> /dev/null` at the end of these.

Could you add this line? Once you have done that, I will approve and we can merge this.